fe/fe-core/src/main/java/org/apache/doris/qe/Coordinator.java

```diff
@@ -318,9 +318,10 @@ private void setFromUserProperty(Analyzer analyzer) {
 
     private void initQueryOptions(ConnectContext context) {
         this.queryOptions = context.getSessionVariable().toThrift();
-        this.queryOptions.setEnableVectorizedEngine(VectorizedUtil.isVectorized());
+        this.queryOptions.setEnableVectorizedEngine(true);
         this.queryOptions.setEnablePipelineEngine(VectorizedUtil.isPipeline());
         this.queryOptions.setBeExecVersion(Config.be_exec_version);
+        this.queryOptions.setEnableVectorizedEngine(true);
```

Comment:
duplicate code.
And I think we should forcibly set `EnableVectorizedEngine` to true, not just set true here.
Because `initQueryOptions(ConnectContext context)` is not always called